cabal.project

```diff
@@ -12,7 +12,7 @@ repository cardano-haskell-packages
 -- The hackage index-state
 index-state: 2023-07-31T10:10:32Z
 -- The CHaP index-state
-index-state: cardano-haskell-packages 2023-08-08T14:32:15Z
+index-state: cardano-haskell-packages 2023-08-28T22:23:10Z
```

Comment:
Run `nix flake lock --update-input CHaP` when updating the `index-state` for CHaP